crown-court-contribution/src/main/java/uk/gov/justice/laa/crime/contribution/service/ContributionService.java

```diff
@@ -177,11 +174,27 @@ public boolean isCds15WorkAround(final RepOrderDTO repOrderDTO) {
                 && InitAssessmentResult.PASS.getResult().equals(initialAssessmentResult);
     }
 
+
     public boolean hasMessageOutcomeChanged(String msgOutcome, RepOrderDTO repOrderDTO) {
         if (null != repOrderDTO) {
             String messageOutcome = Optional.ofNullable(repOrderDTO.getMagsOutcome()).orElse("na");
             return !messageOutcome.equals(msgOutcome);
         }
         return false;
     }
+
+    public boolean hasCCOutcomeChanged(final int repId, final String laaTransactionId) {
+        boolean isOutcomeChanged = false;
+        List<RepOrderCCOutcomeDTO> repOrderCCOutcomeList = maatCourtDataService.getRepOrderCCOutcomeByRepId(repId, laaTransactionId);
+        if (null != repOrderCCOutcomeList && !repOrderCCOutcomeList.isEmpty()) {
+            Optional<RepOrderCCOutcomeDTO> outcomeDTO = repOrderCCOutcomeList.stream().min(Comparator.comparing(RepOrderCCOutcomeDTO::getId));
+            if (outcomeDTO.isPresent()) {
```

Comment:
These two if statements can be merged.

Something like this:
isOutcomeChanged = outcomeDTO.isPresent() && !(null == outcomeDTO.get().getOutcome()
                        || CrownCourtOutcome.AQUITTED.getCode().equals(outcomeDTO.get().getOutcome()));

Reply:
@venkat980 : Thanks. Fixed now